nupic/algorithms/CLAClassifier.py

```diff
@@ -213,6 +213,31 @@ def __setstate__(self, state):
     self._version = BitHistory.__VERSION__
 
 
+  def write(self, proto):
+    proto.id = self._id
+
+    statsProto = proto.init("stats", len(self._stats))
```

Comment:
Since `self._stats` is an array, you'll need to get a python `list`: `proto.stats = self._stats.tolist()` and `self._stats = array.array("f", proto.stats)`